urbackupserver/www2/src/features/status/StatusMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
Nit: I'd have put those files into pages/status/components or similar (as long as they only are used by the page)

Reply:
Agreed, for localizing. Normally, `pages/` or `app/` only contains the actual screens in a React app i.e. anything you put in it is a valid route. 

But I don't think it's required here (yet). Should I add a commit to move content of `features/Status` into `pages/Status` after the final PR is accepted?